README.md

```diff
@@ -17,29 +17,29 @@ Have a look at the [installation documentation](doc/Install.md) for further info
-|:--:|
-|*Frio theme, mobile browser. Timeline and composer view.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-green-profile-1.png?raw=true "Frio theme in desktop browser")
-|*Frio theme, desktop browser. Timeline view, contact info popped up, control menu open.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-green-profile-2.png?raw=true "Frio theme in desktop browser")
-|*Frio theme, desktop browser. Menu open for controlling individual posts.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-red-profile-3.png?raw=true "Frio theme in desktop browser")
-|*Frio theme, desktop browser. Profile view, notification menu open.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-red-profile-2.png?raw=true "Frio theme in desktop browser")
-|*Number of new posts, in total and by group.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-red-profile-1.png?raw=true "Frio theme in desktop browser")
-|*Calender with popup of event.*|
-|![Frio theme default colour in standard browser on tablet](images/screenshots/friendica-frio-default-profile-1.png?raw=true "Frio theme default colour in standard browser on tablet")
-|*Notifications menu and private messages counter, standard browser on tablet.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-brown-profile-2.png?raw=true "Frio theme in desktop browser")
-|*Number of visible contacts, standard browser.*|
-|![Frio theme in desktop browser](images/screenshots/friendica-frio-brown-profile-1.png?raw=true "Frio theme in desktop browser")
-|*Network posts chronologically ordered, standard browser.*|
-|![Vier theme in desktop browser](images/screenshots/friendica-vier-profile.png?raw=true "Vier theme in desktop browser")
-|*Vier theme, desktop browser. Public timeline view.*|
-|![Vier theme in desktop browser](images/screenshots/friendica-vier-community.png?raw=true "Vier theme in desktop browser")
-|*Vier theme, desktop browser. Community post displayed.*|
+| ![Frio theme in mobile browser](images/screenshots/friendica-frio-mobile-profile-1.png?raw=true "Frio theme in mobile browser") ![Frio theme in mobile browser](images/screenshots/friendica-frio-mobile-profile-2.png?raw=true "Frio theme in mobile browser") |
+|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+|                                                                                                    *Frio theme, mobile browser. Timeline and composer view.*                                                                                                    |
```

Comment:
What are the spaces here for? Is this intentional?

Reply:
phpStorm has its own code formatting rules when it comes to Markdown tables.

It caused me problems with the database documentation 😅

Reply:
Have you changed the database documentation by hand? You know that calling `bin/console dbstructure dumpsql > database.sql` does not only create the `database.sql` file but does also update the database documentation?

Reply:
Yes but the system where I edit the code (local laptop) isn’t the same than the system where I have the database (remote VPS) so I thought I could get away with string replacement. In the end I had to use dumpsql but it isn’t convenient.